docs/release.md

```diff
@@ -0,0 +1,22 @@
+# Status AddOn Relesea process 
+
+## Steps to make release
+
+1. Checkout main and fetch from upstream
+2. git checkout <release branch> # e.g. release-0.2
```

Comment:
Do you mean here we are creating a new branch that will be the release branch ?

Reply:
That would be `git checkout -b $release_branch`.

I am not sure what is intended here.

We should have a process that allows adding patches to an old release branch.

Reply:
Most OSS projects  adopts the practice to maintain release branches with major/minor release numbers. In this specific case I am assuming that a release branch was previously created. Typically that should be done at the start of a new release cycle. And yes, we need a process for adding patches, at this point the simple-minded one outlined here was to rebase the latest release branch from main once ready to make a release. There might be other approaches like cherry-picking etc. but this seemed to be the simplest for now.

Reply:
Added also some description/clarifications in the text about this.